gatsby/src/components/Accordion/Accordion.tsx

```diff
@@ -18,12 +23,16 @@ export const Accordion: FC<AccordionProps> = ({
 	defaultOpen = false,
 	showLabel = "Show",
 	hideLabel = "Hide",
+	className,
+	theme,
 }) => {
 	const [isOpen, setIsOpen] = useState(defaultOpen);
+	const themeClass =
+		theme === AccordionTheme.Warning ? styles.warningTheme : "";
 
 	return (
 		<details
-			className={styles.details}
+			className={[styles.details, className, themeClass].join(" ")}
```

Comment:
Might be better using the `classNames` package here to avoid outputting `"undefined"` when `className` empty, as it's optional